src/main/java/pl/jkap/sozzt/config/LoginController.java

```diff
@@ -0,0 +1,13 @@
+package pl.jkap.sozzt.config;
+
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class LoginController {
+
+    @PostMapping("/login")
+    public void login(@RequestBody LoginCredentials credentials) {
+    }
```

Comment:
What this endpoint is supposed to do? It looks like it is waiting for implementation is this true?